ORBIT/phases/design/semi_submersible_design.py

```diff
@@ -106,12 +106,12 @@ def truss_cost(self):
     @property
     def heave_plate_mass(self):
         """
-        Calculates the heave plate mass for a single semi-submersible in tonnes.
-        From original OffshoreBOS model.
+        Calculates the heave plate mass for a single semi-submersible in
+        tonnes. Source: original OffshoreBOS model.
```

Comment:
That's a good point, there are A LOT of spots in the documentation like that. I tried to avoid messing with docstrings more than was required because that's a whole other project, but it could be good to just throw that link there for posterity in all the places where the original BOS model is referenced.

Reply:
Yeah. I'll create an issue to track it.

Reply:
This is addressed in #167